lib/ship_quiet_logistics/sender.rb

```diff
@@ -4,11 +4,11 @@ class Sender
 
     def initialize(bucket)
       @sqs = AWS::SQS.new
-      @queue_name = bucket
+      @queue = bucket
```

Comment:
That's a bit dumb, came with the code dump I did from Wombat's integration, will pass here and there and clean a little.